Rozdział 12/gra2.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Cars</title>
+</head>
+<body>
+
+    <script src="https://code.jquery.com/jquery-2.1.0.js"></script>
+ 
+        <script>
+            
+            var number = 5;
+            var distance = function () {
```

Comment:
+ nie korzystamy widze z tego, ale nie wiem czemu
+ powinno to mieć inną nazwę jako, że jest to funkcja czyli getDistance